entries/i/imgur.com.json

```diff
@@ -2,7 +2,10 @@
   "Imgur": {
     "domain": "imgur.com",
     "contact": {
-      "twitter": "imgur"
+      "twitter": "imgur",
+      "facebook": "imgur",
+      "email": "support@imgur.com",
+      "form": "https://help.imgur.com/hc/en-us/requests/new?ticket_form_id=360003147931"
```

Comment:
Per [the schema](https://github.com/2factorauth/twofactorauth/blob/master/tests/schema.json#L103-L118), only `email` or `form` can be specified, not both. Which would be the best to keep?